staging/src/k8s.io/dynamic-resource-allocation/resourceclaim/pod.go

```diff
@@ -46,3 +46,33 @@ func PodStatusEqual(statusA, statusB []corev1.PodResourceClaimStatus) bool {
 	}
 	return true
 }
+
+func PodExtendedStatusEqual(statusA, statusB *corev1.PodExtendedResourceClaimStatus) bool {
```

Comment:
Unit tests for this, please. Keep this comment open until the new unit tests have been reviewed.